src/cargo/core/package.rs

```diff
@@ -405,13 +405,6 @@ impl<'cfg> PackageSet<'cfg> {
     ) -> CargoResult<PackageSet<'cfg>> {
         // We've enabled the `http2` feature of `curl` in Cargo, so treat
         // failures here as fatal as it would indicate a build-time problem.
-        //
-        // Note that the multiplexing support is pretty new so we're having it
-        // off-by-default temporarily.
-        //
-        // Also note that pipelining is disabled as curl authors have indicated
-        // that it's buggy, and we've empirically seen that it's buggy with HTTP
-        // proxies.
         let mut multi = Multi::new();
```

Comment:
Why is this comment being removed? Does it no longer match the code? Is it out of date?

Reply:
The comment doesn't match the code. Multiplexing is enabled by default (see the line below `multiplexing.unwrap_or(true);`)